lab_3_generate_by_ngrams/main.py

```diff
@@ -143,6 +186,16 @@ def _decode(self, corpus: tuple[int, ...]) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(corpus, tuple) or corpus == ():
```

Comment:
Empty sequences are mapped to False anyway

Reply:
Done